MITADS/Gutenberg_Extractor.py

```diff
@@ -0,0 +1,24 @@
+import requests
+import urllib2
+import time
+from bs4 import BeautifulSoup
+
+urls = open( './books_list.txt', 'r' ).read().splitlines()
```

Comment:
this file where is? can you add it to the pr?

Reply:
I think that should be the final user to write the file.
Doing so the final user can chose wich book add and wich not.
I have included a small explaination how to write the file in the README.md .
I can include more details and a more explicative example how to compile books_list.txt in the readme.

Reply:
This folder will include different exporter for different sources, so we need to specify better.
Also we need a default list, like in the pr :-)